src/app/game-state/item-repo.service.ts

```diff
@@ -562,102 +562,116 @@ export class ItemRepoService {
       value: 1,
       description: 'A basic animal hide.'
     },
-    elmLog: {
-      id: 'elmLog',
-      name: 'elm log',
+    balsaLog: {
+      id: 'balsaLog',
+      name: 'balsa log',
       type: 'wood',
       value: 1,
-      description: 'A terrible quality log.',
+      description: 'A really soft log.',
+    },
+    redwoodLog: {
+      id: 'redwoodLog',
+      name: 'redwood log',
+      type: 'wood',
+      value: 2,
+      description: 'A soft log.',
     },
     cypressLog: {
       id: 'cypressLog',
       name: 'cypress log',
       type: 'wood',
-      value: 2,
+      value: 3,
       description: 'A poor quality log.',
     },
     walnutLog: {
       id: 'walnutLog',
       name: 'walnut log',
       type: 'wood',
-      value: 3,
+      value: 4,
       description: 'An adequate quality log.',
     },
     laurelwoodLog: {
       id: 'laurelwoodLog',
       name: 'laurelwood log',
       type: 'wood',
-      value: 4,
+      value: 5,
       description: 'A nice quality log.',
     },
-    pearwoodLog: {
-      id: 'pearwoodLog',
-      name: 'pearwood log',
+    blackwoodLog: {
+      id: 'blackwoodLog',
+      name: 'blackwood log',
       type: 'wood',
-      value: 5,
+      value: 6,
       description: 'A good quality log.',
     },
     rosewoodLog: {
       id: 'rosewoodLog',
       name: 'rosewood log',
       type: 'wood',
-      value: 6,
+      value: 7,
       description: 'A great quality log.',
     },
+    pearwoodLog: {
+      id: 'pearwoodLog',
+      name: 'pearwood log',
+      type: 'wood',
+      value: 8,
+      description:  'An excellent quality log.',
+    }, 
     zitanLog: {
       id: 'zitanLog',
       name: 'zitan log',
       type: 'wood',
-      value: 7,
-      description: 'An excellent quality log.',
-    },
-    blackwoodLog: {
-      id: 'blackwoodLog',
-      name: 'blackwood log',
-      type: 'wood',
-      value: 8,
+      value: 9,
       description: 'An amazing quality log.',
     },
-    peachwoodLog: {
```

Comment:
No objections to the new wood, but can we add peachwood back in there too? That's an actual historical and mythological thing that I liked.